app/assets/javascripts/mumuki_laboratory/application/avatar.js

```diff
@@ -0,0 +1,28 @@
+mumuki.load(function() {
+  let avatar_id = "";
+
+  $("#edit-avatar-icon").on('click', function(){
+    $('#avatar-picker').modal();
+  });
+
+  $("#user-form").on('submit', function(){
+    if (!avatar_id) {
+      const image_url = $('#user-avatar').attr('src');
+      $(this).append('<input type="hidden" name="user[image_url]" value="' + image_url + '"/>');
+    }
+
+    $(this).append('<input type="hidden" name="user[avatar_id]" value="' + avatar_id + '"/>');
```

Comment:
Creating  the jqueries object wrapper - particularly when selectos are used - may be expensive and cumbersome. I'd recomend to reuse them using `$` prefixed variables.

Reply:
Alright, done